pkgs/applications/networking/cluster/kubectl-node-shell/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "kubectl-node-shell";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "kvaps";
+    repo = "kubectl-node-shell";
+    rev = "v${version}";
+    sha256 = "sha256-dAsNgvHgquXdb2HhLDYLk9IALneKkOxQxKb7BD90+1E=";
+  };
+
+  installPhase = ''
+    install -m755 ./kubectl-node_shell -D $out/bin/kubectl-node_shell
+  '';
+
+  meta = with lib; {
+    description = "Exec into node via kubectl ";
+    homepage = "https://github.com/kvaps/kubectl-node-shell";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ jocelynthode ];
```

Comment:
I assume this PR depends on https://github.com/NixOS/nixpkgs/pull/179375/commits/0eb3c3341b26c4ba1de7fb4cf5e7982da0702421 (#179375)?

Reply:
Yes.

Reply:
I have removed me as maintainer for the time being it's not important anyway.